src/Node/Crypto/Hash.purs

```diff
@@ -33,12 +35,24 @@ hex
   -> Effect String
 hex alg str = hash alg str Hex
 
+hexBuffer
+  :: Algorithm
+  -> Buffer
+  -> Effect String
+hexBuffer alg buf = hashBuffer alg buf Hex
```

Comment:
This is another version shorthand function of `hex`, so i want you to rename `hexBuffer` to `hex'`.